src/functions/uploadToRSIS/application/result-client.ts

```diff
@@ -163,15 +164,21 @@ const mapHTTPErrorToDomainError = (err: AxiosError): Error => {
   return new Error(err.message);
 };
 
-const isRecordAutosaved = (test: StandardCarTestCATBSchema): BooleanAsNumber => {
-  // If the following properties are provided, we safely assume this is not an autosaved record.
-  // As testSummary maybe undefined, we use the non-null assertion operator.
+export const isRecordAutosaved = (test: StandardCarTestCATBSchema): BooleanAsNumber => {
+  const testDate = moment(new Date(test.journalData.testSlotAttributes.start));
+
+  // Two weeks or older is always flagged as autosave
+  if (moment().diff(testDate, 'days') >= 14) {
```

Comment:
Did we not agree that the >=14 days condition should be a part of the conditions below? Or did I recall it incorrectly?

Reply:
We said if anything was older than 14 days we would flag as no write up regardless as it has clearly not been completed.

If for whatever reason we get the test result within the two week window we check the properties if are empty to decide if to set no write up

Reply:
Ah, ok - that's fine, just wanted to confirm 👍